zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -77,10 +77,26 @@ class MeasurementType(enum.IntFlag):
             attr=ElectricalMeasurement.AttributeDefs.rms_current.name,
             config=REPORT_CONFIG_OP,
         ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.rms_current_ph_b.name,
+            config=REPORT_CONFIG_OP,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.rms_current_ph_c.name,
+            config=REPORT_CONFIG_OP,
+        ),
         AttrReportConfig(
             attr=ElectricalMeasurement.AttributeDefs.rms_current_max.name,
             config=REPORT_CONFIG_DEFAULT,
         ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_b.name,
+            config=REPORT_CONFIG_DEFAULT,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.rms_current_max_ph_c.name,
+            config=REPORT_CONFIG_DEFAULT,
+        ),
```

Comment:
The not so great thing about this is that we'll add another four attributes to poll per device on each startup. 
With https://github.com/zigpy/zha/pull/324, that makes it eight attributes that are polled (per device on startup).

That isn't ideal, so we might want to look at only polling the attributes [here](https://github.com/zigpy/zha/blob/fd6cf2f570f1778a9c0933a56ebf476caafe859d/zha/zigbee/cluster_handlers/__init__.py#L458-L460) if they're not marked as unsupported already. Unsupported attributes might change with firmware updates, but otherwise shouldn't (hopefully).

Reply:
Ah never mind, this PR includes the changes from the other one.

Reply:
But we'd also need to set up attribute reporting for voltage and power factor here, right?

Reply:
should we add support for the conditional polling before this PR gets in?

Reply:
The other PR was merged without it too, so this one is also fine, but we should try and target to try implementing that before the next HA beta (/ZHA release).